nav2_costmap_2d/src/clear_costmap_service.cpp

```diff
@@ -112,6 +112,15 @@ void ClearCostmapService::clearRegion(const double reset_distance, bool invert)
       clearLayerRegion(costmap_layer, x, y, reset_distance, invert);
     }
   }
+
+  layers = costmap_.getLayeredCostmap()->getFilters();
+
+  for (auto & layer : *layers) {
+    if (layer->isClearable()) {
```

Comment:
https://github.com/ros-planning/navigation2/blob/2115e26849105fde9322879bb4a25790234f95d1/nav2_costmap_2d/include/nav2_costmap_2d/costmap_filters/costmap_filter.hpp#L125 this is always false for all costmap filters

Reply:
Changed in https://github.com/ros-planning/navigation2/pull/2308/commits/a9f92d8711e0bd50fe1bd3b54f2c9304c3f87b01#diff-f2ea1827ae80319b8a5efca999c9f81e1a2cf95578e516ea48aa4e606c32cd1eL116-R117